jerry-ext/handler/handler-gc.c

```diff
@@ -31,6 +31,13 @@ jerryx_handler_gc (const jerry_value_t func_obj_val, /**< function object */
   (void) args_p; /* unused */
   (void) args_cnt; /* unused */
 
-  jerry_gc ();
+  jerry_gc_mode_t mode = JERRY_GC_SEVERITY_LOW;
+
+  if (args_cnt > 0 && jerry_value_to_boolean (args_p[0]))
+  {
+    mode = JERRY_GC_SEVERITY_HIGH;
+  }
```

Comment:
This all could go into a `?:` expression to avoid making the initialization of `mode` potentially a dead expression.

```c
  jerry_gc_mode_t mode = args_cnt > 0 && jerry_value_to_boolean (args_p[0]) ? JERRY_GC_SEVERITY_HIGH
                                                                            : JERRY_GC_SEVERITY_LOW;
```

Reply:
BTW, the doc string of the function should be updated describing how it deals with its arguments. Something like:

The routine invokes the GC with high severity if it has at least one argument and the first argument is equivalent to boolean true; it calls the GC with low severity otherwise.